apps/buildog/src/app/organizations/page.tsx

```diff
@@ -0,0 +1,137 @@
+                Add Organization
+              </Button>
+            </DialogTrigger>
+            <DialogContent>
+              <DialogHeader>
+                <DialogTitle>Add New Organization</DialogTitle>
+                <DialogDescription>Enter the name of your new organization.</DialogDescription>
+              </DialogHeader>
+              <div className="space-y-4 py-4">
+                <div>
+                  <Label htmlFor="orgName">Organization Name</Label>
+                  <Input
+                    id="orgName"
+                    value={newOrgName}
+                    onChange={(e) => setNewOrgName(e.target.value)}
+                    placeholder="My Unique Organization"
+                  />
+                </div>
+              </div>
+              <DialogFooter>
+                <Button onClick={handleCreateOrganization} className="w-full">
+                  Create Organization
+                </Button>
+              </DialogFooter>
+            </DialogContent>
+          </Dialog>
+        </div>
+      ) : (
+        // Organizations present case
+        <div className="w-full">
+          <div className="flex items-center justify-between w-full mb-6">
+            <h2 className="text-2xl font-semibold">Your Organizations</h2>
+            <Dialog>
+              <DialogTrigger asChild>
+                <Button size="lg">Add Organization</Button>
+              </DialogTrigger>
+              <DialogContent>
+                <DialogHeader>
+                  <DialogTitle>Add New Organization</DialogTitle>
+                  <DialogDescription>Enter the name of your new organization.</DialogDescription>
+                </DialogHeader>
+                <div className="space-y-4 py-4">
+                  <div>
+                    <Label htmlFor="orgName">Organization Name</Label>
+                    <Input
+                      id="orgName"
+                      value={newOrgName}
+                      onChange={(e) => setNewOrgName(e.target.value)}
+                      placeholder="My Unique Organization"
+                    />
+                  </div>
+                </div>
+                <DialogFooter>
+                  <Button onClick={handleCreateOrganization} className="w-full">
+                    Create Organization
+                  </Button>
+                </DialogFooter>
+              </DialogContent>
+            </Dialog>
+          </div>
+
+          {/* Organization cards grid */}
+          <div className="w-full max-w-6xl grid grid-cols-1 sm:grid-cols-2 gap-6 mt-10">
+            {organizations.map((org) => (
+              <Card
+                key={org.id}
+                className="p-6 cursor-pointer h-36 w-full max-w-lg flex items-center justify-between border rounded-lg shadow-md hover:shadow-lg transition-shadow duration-300 ease-in-out"
+                onClick={() => router.push(`/organizations/${org.name}/blog`)}
```

Comment:
- we can directly send the organization main page. We can add some analytics in the future

- Should we use name or Id here? I saw that you create an id also in dummy data. I think we can manage these with ids because it is unique. After changing this, you need to change folder name [organizationName] to [organizationId]

```suggestion
                onClick={() => router.push(`/organizations/${org.id}/`)}
```